packages/next/server/view-render.tsx

```diff
@@ -216,16 +216,18 @@ export async function renderToHTML(
     ComponentMod,
   } = renderOpts
 
+  const isFlight = query.__flight__ !== undefined
+  const flightRouterPath = isFlight ? query.__flight_router_path__ : undefined
```

Comment:
Do we want to ensure these values are cleared from the query before passing to `getServerSideProps` similar to the current render handling https://github.com/vercel/next.js/blob/6aa9ef1955b16a5e2a9e624bfc7e61595f5cfc11/packages/next/server/render.tsx#L553?